Hedvig/Assets/Localization/Localization.swift

```diff
@@ -1180,6 +1188,42 @@ struct Translations {
                     return """
                 Hedvig tror starkt på open-source, här finner du en lista och tillhörande licenser för de biblioteken vi förlitar oss på 💕
                 """
+
+                case .LOGOUT_BUTTON:
+                    if let text = TranslationsRepo.find(.LOGOUT_BUTTON) {
+                        return text
+                    }
+
+                    return """
+                Logga ut
+                """
```

Comment:
Why so many quotation marks?

Reply:
This one is auto-generated, this is the swift way of handling multiline text hehe